### PR TITLE
Day 12 pull request

### PR DESCRIPTION
**Pull request by tuanbui0509**
please check ...

### CODE REVIEW

File: Day_11/Push_a_branch.md [hunk 1]
```diff
@@ -5,4 +5,7 @@
 - 2. git push origin (branch)
 - 3. create a pull request on Github
 - 4. review code
+    - 4.1 review code online (Github)
```
Comment (buingoctuan99) [inline Day_11/Push_a_branch.md]: remove 3 line 8,9,10

**Review by buingoctuan99 — COMMENTED** [review]
okie

**Review by buingoctuan99 — COMMENTED** [review]
nothing

File: Day_11/Push_a_branch.md (hunk 1 above)
Comment (tuanbui0509) [reply Day_11/Push_a_branch.md]: thank you very much

**Pull request merged**